### PR TITLE
Update Auth0.swift docs [SDK-1631]

### PR DESCRIPTION
**Pull request by Widcket**
### Changes

- Fixed old/broken links
- Updated code snippets
- Mentioned new features (Refresh Token Rotation and WebAuth for macOS)
- Fixed casing mishaps

### CODE REVIEW

**Review by adamjmcgrath — DISMISSED** [review]

File: articles/users/references/link-accounts-server-side-scenario.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ router.get("/", async (req, res) => {
     let getUsersWithSameVerifiedEmail = [];
     const getUserProfile = auth0Client.getUser(sub);
     if (email_verified)
-      // account linking is only offered verified email
+      // account linking isx only offered verified email
```
Comment (lbalmaceda) [inline articles/users/references/link-accounts-server-side-scenario.md]:
```suggestion
      // account linking is only offered verified email
```

File: articles/users/references/link-accounts-server-side-scenario.md [hunk 2]
```diff
@@ -58,29 +58,28 @@ router.get("/", async (req, res) => {
...
-      });
-    }
-    ```
+  ```js
+  const request = require('request');
+  class Auth0Client {
+    ...
+    async getUsersWithSameVerifiedEmail({ sub, email }) {
+      return await this.request({
+        url: `${process.env.ISSUER_BASE_URL}/api/v2/x`x``users`,
```
Comment (lbalmaceda) [inline articles/users/references/link-accounts-server-side-scenario.md]:
```suggestion
        url: `${process.env.ISSUER_BASE_URL}/api/v2/users`,
```

File: articles/users/guides/email-verified.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+title: Email Verified Usage
```
Comment (lbalmaceda) [inline articles/users/guides/email-verified.md]: This is not a Swift article. Are you sure this file belongs to this PR? Maybe a leftover from a rebase?

**Review by lbalmaceda — CHANGES_REQUESTED** [review]

File: articles/users/guides/email-verified.md (hunk 3 above)
Comment (Widcket) [reply articles/users/guides/email-verified.md]: You're right, this file does not belong to this PR. Yes, a leftover from the rebase.

Comment (Widcket) [reply articles/users/guides/email-verified.md]: It's been corrected now.

**Review by lbalmaceda — APPROVED** [review]
LGTM

**Pull request merged**